_data/harnesses/splitting/kstreams.yml

```diff
@@ -174,3 +174,9 @@ prod:
           file: tutorial-steps/dev/clean-up.sh
           render:
             skip: true            
+
```

Comment:
This one is missing the `ccloud:` and `steps`

Reply:
@bbejeck good catch, thanks, addressed.